src/jit/emit.h

```diff
@@ -923,11 +925,12 @@ class emitter
 
             struct
             {
-                regNumber _idReg3 : REGNUM_BITS;
-                regNumber _idReg4 : REGNUM_BITS;
 #ifdef _TARGET_ARM64_
-                unsigned _idReg3Scaled : 1; // Reg3 is scaled by idOpSize bits
+                emitLclVarAddr iiaLclVar;
```

Comment:
The basic idea is that `idAddrUnion` is `ptrsize_t`.  `iiaLclVar` is constrained to be 32 bits for 32 bit platforms.  Therefore on a 64 bit platform these can both be in use simultaneously.